assets/src/edit-story/components/panels/backgroundSizePosition.js

```diff
@@ -68,6 +68,7 @@ function BackgroundSizePositionPanel({ selectedElements, pushUpdate }) {
         <FlipControls
           onChange={(value) => pushUpdate({ flip: value }, true)}
           value={flip}
+          margin={'36px'}
```

Comment:
Any way we can avoid passing down a specific style instruction?

Or at least call this `elementSpacing` or something?

Reply:
Renamed.